infrastructure/smart-contracts/contracts/drops/TinyKeysAirdrop.sol

```diff
@@ -102,6 +102,7 @@ contract TinyKeysAirdrop is Initializable, AccessControlUpgradeable {
     
     function processAirdropSegmentOnlyMint(uint256 _qtyToProcess) external onlyRole(DEFAULT_ADMIN_ROLE)  {
         require(airdropStarted, "Airdrop not started");
+        require(!airdropEnded, "Airdrop already completed");
```

Comment:
Lets combine these into a single require statement. Is more simple and will save bytecode size.

Reply:
Lets do it, pls also update the testcase, it expects the hardcoded error message I belive